frontend/src/metabase/querying/notebook/utils/steps.ts

```diff
@@ -104,7 +115,18 @@ const STEPS: NotebookStepDef[] = [
     // NOTE: summarize is a combination of aggregate and breakout
     type: "summarize",
     clauseType: "aggregation",
-    valid: query => {
+    valid: (query, stageIndex) => {
+      // do not suggest summarizing if there is aggregation without breakout on the last stage
+      if (stageIndex >= 1) {
```

Comment:
Maybe extract to a util here?

Reply:
fixed e810c883d1fa13de83e310d7a7fab6ae038cfdae